src/sw.ts

```diff
@@ -42,6 +42,7 @@ interface StoreReponseInCacheOptions {
  */
 interface LocalSwConfig {
   installTimestamp: number
+  originIsolationWarningAccepted: boolean
```

Comment:
this should probably be in private config, like https://github.com/ipfs/service-worker-gateway/blob/957a9506c6c89b069d27ca4f6986e9ee1c6ef4a3/src/lib/config-db.ts#L19-L21

And then we could just query that on the client instead of using sessionStorage